src/detect.h

```diff
@@ -127,6 +127,10 @@ enum DetectSigmatchListEnum {
 
     DETECT_SM_LIST_BASE64_DATA,
 
+    DETECT_SM_LIST_DNP3_DATA_MATCH,
+    DETECT_SM_LIST_DNP3_MATCH,
```

Comment:
can DETECT_SM_LIST_DNP3_MATCH and DETECT_SM_LIST_DNP3_LUA_MATCH be merged into a single list?